src/integrationTest/java/uk/nhs/hee/trainee/details/TestJwtUtil.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright 2022 Crown Copyright (Health Education England)
```

Comment:
📅 😀

Reply:
I knew you'd spot that, technically it's duplicated from the unit test module... so it's kinds right? (Once other tests are moved I think the unit test one gets deleted)